### DISCUSSION TITLE
mspider/user-guide/traffic-governance/vsparams/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1269] [Announcements]
# mspider/user-guide/traffic-governance/vsparams/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/user-guide/traffic-governance/vsparams/

**Comment by liyuerich** [comment]
版本服务	spec.http.-name.route.-destination.subset	可选。列表内容来自所选服务的可用目标规则。

版本服务 - 应该为 “服务版本”
